libPhoneNumber/NBPhoneNumberOfflineGeocoder.h

```diff
@@ -0,0 +1,40 @@
+//
+//  NBPhoneNumberOfflineGeocoder.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "NBGeocoderMetadataHelper.h"
+#import "NBPhoneNumber.h"
```

Comment:
Even though the class is used here. Use forward declaration instead:

`@class NBPhoneNumber;`

before the `@interface` declaration.

This is the same thing in C++ where you don't want to include too many headers in the .h file

Reply:
Yup, fixed.